tests/EpisodeTest.php

```diff
@@ -4,36 +4,311 @@
+
+        $this->assertTrue(is_array($episodes));
+        $this->assertArrayHasKey('episode_name', $episodes['data'][0]);
+        $this->assertArrayHasKey('episode_description', $episodes['data'][0]);
+    }
+
+    /**
+     * Assert that EpisodeRepository's  createEpisode adds a new
+     * episode into the database.
+     *
+     * @return void
+     */
+    public function testCreateEpisode()
+    {
+        $repository = new EpisodeRepository();
+        $episode = $repository->createEpisode([
+            'episode_name' => 'Swanky new Episode',
+            'episode_description' => 'Swanky New episode description',
+            'view_count'            => 10,
+            'image'                 => "http://goo.gl/8sorZR",
+            'audio_mp3'             => "http://goo.gl/LkNP5M",
+            'channel_id'            => 1,
+            'status'                => 0,
+            'likes'                 => 10
         ]);
 
-        $this->seeInDatabase('episodes', [
-            'episode_name'          => 'test',
-            'episode_description'   => 'test',
-            'channel_id'            => $channel->id,
-            'view_count'            => 0,
-            'image'                 => 'dummy url',
-            'audio_mp3'             => 'dummy url',
-            'status'                => 0
+        $repository = new EpisodeRepository();
+        $getEpisode = $repository->findEpisodeById($episode['id'])->toArray();
+
+        $this->assertTrue(is_array($getEpisode));
+        $this->assertArrayHasKey('episode_name', $getEpisode);
+        $this->assertArrayHasKey('episode_description', $getEpisode);
+        $this->seeInDatabase('episodes', ['episode_name' => 'Swanky new Episode']);
+    }
+
+    /**
+     * Assert that EpisodeRepository's updateEpisode updates an existing episode
+     * @return [type] [description]
+     */
+    public function testUpdateEpisode()
+    {
+        $repository = new EpisodeRepository();
+        $episode = $repository->createEpisode([
+            'episode_name' => 'Swanky new Episode',
+            'episode_description' => 'Swanky New episode description',
+            'view_count'            => 10,
+            'image'                 => "http://goo.gl/8sorZR",
+            'audio_mp3'             => "http://goo.gl/LkNP5M",
+            'channel_id'            => 1,
+            'status'                => 0,
+            'likes'                 => 10
         ]);
+        $update = $repository->updateEpisode($episode['id'], 'episode_name', 'Swanky updated name');
+        $this->assertTrue($update);
+
+        $getEpisode = $repository->findEpisodeById($episode['id'])->toArray();
+
+        $this->assertTrue(is_array($getEpisode));
+        $this->assertArrayHasKey('episode_name', $getEpisode);
+        $this->assertArrayHasKey('episode_description', $getEpisode);
+        $this->seeInDatabase('episodes', ['episode_name' => 'Swanky updated name']);
+    }
+    /**
```

Comment:
**Code Quality - Nitpick**

There should be a newline space at least between a method and another method. Right now, the comments of the `testActiveAndPendingEpisodes` are directly at the bottom of the previous function.